config/routes.rb

```diff
@@ -195,6 +195,11 @@
         get '/:id/institution/edit' => 'degrees/institution#edit', as: :edit_degree_institution
         patch '/:id/institution/edit' => 'degrees/institution#update'
 
+        get '/:id/naric_statement' => 'degrees/naric_statement#new', as: :degree_naric_statement
+        post '/:id/naric_statement' => 'degrees/naric_statement#create'
+        get '/:id/naric_statement/edit' => 'degrees/naric_statement#edit', as: :edit_degree_naric_statement
+        patch '/:id/naric_statement/edit' => 'degrees/naric_statement#update'
```

Comment:
We try to avoid underscores in URLs if we can:

```suggestion
        get '/:id/naric_statement' => 'degrees/naric-statement#new', as: :degree_naric_statement
        post '/:id/naric_statement' => 'degrees/naric-statement#create'
        get '/:id/naric_statement/edit' => 'degrees/naric-statement#edit', as: :edit_degree_naric_statement
        patch '/:id/naric_statement/edit' => 'degrees/naric-statement#update'
```